cogs/music.py

```diff
@@ -966,6 +970,25 @@ async def get_song_info(self) -> str:
 		return f"{self.song.title} - {self.song.url}"
 
 
+def validate_url(url):
+	"""
+	Checks to see if url has any valid extractors for youtube_dl
+
+	Parameters:
+		- url: the url to search for extractors.
+
+	Returns:
+		-True, if site has dedicated extractor
+		-False, if site has no dedicated extractor
+	"""
+	extractors = youtube_dl.extractor.gen_extractors()
```

Comment:
We're expecting a youtube link, so we can get the youtube extractor directly with `youtube_dl.extractor.get_info_extractor('Youtube')`. Then, we can just `return e.suitable(url)`.